packages/messaging/lib/index.d.ts

```diff
@@ -653,7 +729,7 @@ export namespace FirebaseMessagingTypes {
      *
      * @param options Options to override senderId (iOS) and projectId (Android).
      */
-    deleteToken(options?: TokenOptions): Promise<void>;
+    deleteToken(options?: DeleteTokenOptions): Promise<void>;
```

Comment:
again assuming `NativeTokenOptions` existed, to give a home to the native-only appName/senderId stuff

```suggestion
    deleteToken(options?: NativeTokenOptions): Promise<void>;
```